gpytorch/mlls/variational_elbo.py

```diff
@@ -29,8 +29,14 @@ def forward(self, variational_dist_f, target, **kwargs):
 
         log_likelihood = self.likelihood.variational_log_probability(variational_dist_f, target, **kwargs).div(
             num_batch
-        )
-        kl_divergence = torch.distributions.kl.kl_divergence(variational_dist_u, prior_dist).sum().div(self.num_data)
+        ).sum()

```

Comment:
I think we need to divide by `num_batch` here?

Reply:
Or rather divide by the batch size

Reply:
Sorry - the git diff threw me off here. This is totally fine.

Reply:
`num_data` is correct!